content/install.md

```diff
@@ -1,93 +1,11 @@
-# Install the Wazo Platform UC use case
+# Install the Wazo Platform
 
-## Requirements
+Wazo Platform is an Open Source project allowing to build carrier grade programmable IP communication infrastructures. You can pick and choose the components you need to build your infrastructures with class 5 features like audio and video calls, chat, call centers, conferences, voicemail, etc., and class 4 features like security, routing, load balancing, etc. You can augment the platform and integrate it with your subscription and billing capabilities or any value-added services like Speech to Text, A.I. or sentiment analysis. These communication infrastructures can be deployed and scaled on bare metal, virtual machines or containers.
 
-- Memory: 2 GiB of memory is a tight minimum
-- Storage: 2.5 GiB of storage is a very tight minimum, 8 GiB is comfortable
+## Unified Communication
 
-## Procedure
+Follow [our guide to install the Wazo Platform Unified Communication engine](https://wazo-platform.org/install/unified-communication).
```

Comment:
Please use a relative link to allow to work locally and in CI.

Also we should not add anything after Wazo Platform to be seen as a sub-product. We should describe it as a use case of Wazo Platform like "our guide to install the Unified Communication all-in-one use case".